build-container/entrypoint.sh

```diff
@@ -18,7 +20,14 @@ if [ ! -z "${FAILOVER_RELEASE_NAME}" ]; then
     FAILOVER_REPO_DIR="$(mktemp -d)"
     cd "${FAILOVER_REPO_DIR}"
     FAILOVER_BASE_URL="https://github.com/archzfs/archzfs/releases/download/${FAILOVER_RELEASE_NAME}"
-    if ! curl -fsL "${FAILOVER_BASE_URL}/archzfs.db.tar.xz" | tar xvJ; then
+    db_file="archzfs.db.tar.xz"
+    curl -f -o "${db_file}" -L "${FAILOVER_BASE_URL}/${db_file}"
+    curl -f -o "${db_file}.sig" -L "${FAILOVER_BASE_URL}/${db_file}.sig"
```

Comment:
Perhaps we want to keep these curl executions in an `if !` like the next steps? Something like "Couldn't fetch failover..."?